testdata/case/src/github.com/sgtest/godep-exclude

```diff
@@ -1 +0,0 @@
-Subproject commit bbee76ddfccee783ca1ee5bb9850f1d6475324ae
```

Comment:
this commit has the same title as the next one, but this one seems unrelated to the message `removed unused srcfileConfig.env`